### ISSUE TITLE
How to run test/database_events/sync_performer_test.rb ?

### ISSUE THREAD

**Issue by 10io** [tomykaira/clockwork#157]
Hello there!

I'm trying to add a new feature to the options accepted by `sync_database_events` but before that, I'd like to be able to run all the existing tests.

This might be a dumb question but how do you run `test/database_events/sync_performer_test.rb` ?
I tried with
```shell
$ bundle exec ruby -Itest test/database_events/sync_performer_test.rb 
```
but nothing is executed. 

I also tried setting the var env `BUNDLE_GEMFILE` to `./gemfiles/activesupport3.gemfile` or `./gemfiles/activesupport4.gemfile` but I got the same result.

Inspecting the code, I'm not sure how these tests are meant to be executed. Can someone provide me some light?

**Comment by robisenberg** [comment]
This is now fixed after #174. Apologies for the long wait.

**Comment by 10io** [comment]
np, thanks!
